grafana/PostgreSQLDetails.json

```diff
@@ -846,28 +845,74 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(rate(ccp_stat_database_deadlocks{job=\"[[jobname]]\"}[5m]))",
+          "expr": "sum(ccp_locks_count{job=\"[[jobname]]\",mode=\"accessexclusivelock\"})",
           "format": "time_series",
           "intervalFactor": 2,
-          "legendFormat": "Conflicts",
-          "metric": "pg_stat_database_conflicts",
+          "legendFormat": "accessexclusive",
           "refId": "A",
           "step": 240
         },
         {
-          "expr": "sum(rate(ccp_stat_database_conflicts{job=\"[[jobname]]\"}[5m]))",
```

Comment:
may it is git or github but at this level this change looks questionable.

Reply:
VSCode auto formats json on save.

Reply:
Looks like this is because the entries are just re-ordered. It's added back later on a different line.